src/Akka.Persistence.Sql.Linq2Db/Query/Dao/BaseByteReadArrayJournalDAO.cs

```diff
@@ -126,32 +126,22 @@ private static int MaxTake(long max)
             var maxTake = MaxTake(max);
             return AsyncSource<JournalRow>.FromEnumerable(new{separator,tag,offset,maxOffset,maxTake,_connectionFactory},
                     async(input)=>
-                {
+                    {
+                    var tag = $"{separator}{input.tag}{separator}";
                     using (var conn = input._connectionFactory.GetConnection())
                     {
                         return await conn.GetTable<JournalRow>()
-                            .Where<JournalRow>(r => r.tags.Contains(input.tag))
+                            .Where<JournalRow>(r => r.tags.Contains(tag))
                             .OrderBy(r => r.ordering)
                             .Where(r =>
                                 r.ordering > input.offset && r.ordering <= input.maxOffset)
                             .Take(input.maxTake).ToListAsync();
                     }
-                }).Via(perfectlyMatchTag(tag, separator))
```

Comment:
We do not need this check anymore because we are sure that the tag is matched perfectly